Adds Initial Set of Documents (#3)
ArlingtonCouncilOfPTAs/tech-guide: online-bookkeeping.md

## Online Bookkeeping Tools
There are many options available for PTAs to track finances.  These range from homemade spreadsheets to professional tools such as QuickBooks or Sage 50.  Most PTAs are best served by something that falls in between these two extremes.  

> This list is limted to cloud-based software options that have recieved positive reviews from PTAs in the district.  All of these solutions support web-based access, including read-only accounts for non-signers to independantly review transactions.

Reviewer: ```suggestion
This list is limited to cloud-based software options that have received positive reviews from PTAs in the district.  All of these solutions support web-based access, including read-only accounts for non-signers to independently review transactions.
```

### myPTEZ
[myPTEZ](https://www.myptez.com) is a membership and finance tracking software designed specifically for PTAs.  The financial tools in myPTEZ are more basic, but have the benefit of being integrated with member reporting to Texas PTA.  It includes budgeting, maintaining the checkbook register, and an a simple online store.  

[Pricing](https://www.myptez.com/join/) depends on the amount of money your PTA takes in during the year.  A 45 day free trial is available.

Visit the [myPTEZ YouTube channel](https://www.youtube.com/channel/UCDt1o46QYUcbgL2fQItMdQw/videos) for more information.

### MoneyMinder

> [MoneyMinder](https://www.moneyminder.com) supports the standard suite of budgeting and financial reporing tools, as well as a simple online store.  MoneyMinder is designed for smaller volunteer organizations, like a PTA.

Reviewer: ```suggestion
[MoneyMinder](https://www.moneyminder.com) supports the standard suite of budgeting and financial reporting tools, as well as a simple online store.  MoneyMinder is designed for smaller volunteer organizations, like a PTA.
```

For an additional fee, it also has the ability to import bank transactions, if your bank supports that feature.  The financial features of MoneyMinder cost [$179/year](https://moneyminder.com/pricing/).  A [30-day free trial](https://moneyminder.com/free-trial/) is available.

Visit the [MoneyMinder YouTube channel](https://www.youtube.com/c/MoneyMinder/videos) for more information.

### Wave
[Wave](https://www.waveapps.com) is a [free](https://www.waveapps.com/pricing) general-purpose bookkeeping software.  It include a full suite of bookkeeping and financial reporting tools, but does not include budgeting or an online store.  Wave can sync with your bank account, if the bank supports that.

Visit the [WaveApps YouTube channel](https://www.youtube.com/user/WaveAccounting/videos) for more information.

> [<- back](./README.md)

Reviewer: ```suggestion
[<- Go Back](./README.md)
```
